### DISCUSSION TITLE
Is there any way to get capacity fade values without cycling experiments?

### DISCUSSION THREAD

**Discussion by cyasing** [pybamm-team/PyBaMM#2935] [Q&A] [answered]
The only capacity related variables I could find were in solution summary variables which can be only accessed through cycling. While I do know there wouldn't be a considerable capacity fade in a short duration, I am still interested in measuring it. Is there no other variable or workaround for it?

**Comment by brosaplanella** [comment, marked as answer]
What summary variables are you interested in? Note that the solution already has some capacity variables. You can find them by using the `model.variables.search("A.h")` command, which returns
```
Discharge capacity [A.h]
Loss of capacity to SEI [A.h]
Loss of capacity to SEI on cracks [A.h]
Loss of capacity to lithium plating [A.h]
Negative electrode capacity [A.h]
Positive electrode capacity [A.h]
Throughput capacity [A.h]
Total capacity lost to side reactions [A.h]
Total lithium capacity [A.h]
Total lithium capacity in particles [A.h]
```

You can then plot them as ususal.

**Reply by cyasing** [reply]
Whoa, for some reason I had missed these. Thanks!
